mysql-test/suite/rpl/t/percona_bug_ps6979_showprocesslist.test

```diff
@@ -0,0 +1,21 @@
+--source include/master-slave.inc
+
+connection master;
```

Comment:
Even though I'm not an official reviewer, here are my two cents.

1. You may remove this line as `master-slave.inc`` by default leaves the connection to master.
2. Consider using `rpl_connection_master.inc` and `rpl_connection_slave.inc` instead of just `connection` mysqltest command. Using `rpl_connection_master.inc` will print `[connection master]` into the result file and makes the result file more readable.